build.cake

```diff
@@ -88,7 +126,14 @@ Task("Build")
                 .WithTarget("Build")
                 .WithProperty("Configuration", configuration)
                 .SetVerbosity(Verbosity.Minimal));
-	    }
+        }
+
+        version = context.EnvironmentVariable("GitVersion_MajorMinorPatch");
+        var nugetVersion = context.EnvironmentVariable("GitVersion_NuGetVersion");
+        var preReleaseTag = context.EnvironmentVariable("GitVersion_PreReleaseTag");
+        var semVersion = context.EnvironmentVariable("GitVersion_LegacySemVerPadded");
```

Comment:
None of this is having any effect so far. I think you need to get these variables in the Package task.